src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +730,62 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export', cls=LazyConfigureGroup)
```

Comment:
When running `verdi computer export --help`, it still shows the `core.local` and `core.ssh` options usually shown when running `verdi computer configure --help`. This should come from the `LazyConfigureGroup`, and can be resolved by just removing this argument:

```suggestion
@verdi_computer.group('export')
```